### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by peitalin** [avocode/nachos-ui#23]
Installing this package breaks react-native@^47:
This is the error that pops up:
```
error: bundling failed: "Unable to resolve module `AccessibilityInfo` from `$ROOT/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map\n\nThis might be related to https://github.com/facebook/react-native/issues/4968\nTo resolve try the following:\n  1. Clear watchman watches: `watchman watch-del-all`.\n  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.\n  3. Reset packager cache: `rm -fr $TMPDIR/react-*` or `npm start -- --reset-cache`."
```

Uninstalling nachos-ui DOES NOT fix this error.

**Comment by mattapperson** [comment]
This issue is being tracked now in #34. Closing.
